docs/proposals/excluding-drives.md

```diff
@@ -0,0 +1,108 @@
+# Proposal: Specific drives excluding
+
+Last updated: 07.02.2022
+
+
+## Abstract
+
+CSI Baremetal needs to have API for excluding specific drives from the common storage pool. 
+Excluded drives won't be used for allocating new Volumes or LVGs.
+
+## Background
+
+CSI customers might not want to use some disks for whatever reason:
+- for another application not managed by CSI
+- as a workaround for any disk detection issues in CSI
+
+So in these cases CSI haven't to allocate Volumes for some drives.
+
+## Proposal
+
+### User API
+
+The rules to include/exclude drives based on:
+- Size
+- Type (HDD, SSD, ...)
+- Serial Number
+- PID
+- VID
+```yaml
+drive_rule:
```

Comment:
Use `camelCase` not `snake_case` please.

Reply:
Fixed